peakdet/operations.py

```diff
@@ -8,7 +8,6 @@
 from scipy import interpolate, signal
 from peakdet import editor, utils
 
-
 @utils.make_operation()
```

Comment:
This is a minor style thing, but we do add two lines between imports and the first function (including decorators!)